src/api/RecognitionService.tsx

```diff
@@ -0,0 +1,11 @@
+import MainApiProtected from "./MainAPIProtected";
+
+export default class RecognitionService extends MainApiProtected {
+    public constructor() {
+        super();
+    }
+    public getFeed = () => {
+        console.log("Got feed");
+        return this.instance.get('/recognitions/all');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Benedict-Scrumberbatch/peer-recognition/pullRequest?prid=7204339)
```suggestion
        return this.instance.get("/recognitions/all");
```